src/helpers/userUtils.ts

```diff
@@ -0,0 +1,8 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export const getUser = (userId: number): User | null => {
```

Comment:
Use more descriptive name.
```suggestion
export const getUserById = (userId: number): User | null => {
```